### DISCUSSION TITLE
Unable to enable Actions for this repository

### DISCUSSION THREAD

**Discussion by atocqueville** [community/community#26931] [Actions]
<p>Hello,</p>
<p>I registered for the Actions beta about 2 weeks ago and everything worked perfectly.</p>
<p>Now that I’ve finished one of my repo and transfered the Action to my org repo, I can’t run GitHub Actions on one of my repos again !</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/f/fca90407052a91bef66a0581266d0e529c3720de.png" data-download-href="/uploads/short-url/A38r6aAtQkVBLhSqSEUVknM4KUm.png?dl=1" title="Screen Shot 2019-10-29 at 11.10.35 (2).png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/11229127/181100835-37aa5caf-611f-4efb-acdc-df745dee7f4c.png" alt="Screen Shot 2019-10-29 at 11.10.35 (2).png" data-base62-sha1="A38r6aAtQkVBLhSqSEUVknM4KUm" width="690" height="258" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/f/fca90407052a91bef66a0581266d0e529c3720de_2_690x258.png, https://aws1.discourse-cdn.com/github/optimized/2X/f/fca90407052a91bef66a0581266d0e529c3720de_2_1035x387.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/f/fca90407052a91bef66a0581266d0e529c3720de_2_1380x516.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/f/fca90407052a91bef66a0581266d0e529c3720de_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2019-10-29 at 11.10.35 (2).png</span><span class="informations">3353×1255 217 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>When I click on ‘Enable Actions on this repository’, i get the error message in blue on top of the screen: ‘Unable to enable Actions for this repository’.</p>
<p>This is not repo-specific as this is the 4th repository I tried on.</p>
<p>I have no idea what to do, do you have any ?</p>

**Comment by gabac** [comment]
<p>I have the same issue…</p>
<p>Do you see this with a private repo?</p>

**Comment by mscoutermarsh** [comment]


**Reply by jmkjoel004** [reply]
yes it still happening

**Reply by stifflerjoe** [reply]
It is happening

**Comment by ramohinternet** [comment]
<p>Hey,</p>
<p>This is happening for me today. I have been using Github APIs to create repo, secrets. and I have workflow files in the repo.</p>
<p>when I put multiple branches on <code>on.push.branches</code> then it gives me the message: “Workflows aren’t being run on this repository” and " Actions are not currently enabled on this repository."</p>
<p>but infact it works when there is only single branch specified in <code>on.push.branches</code></p>
<p>Take a look at this repo. and actions: <a href="https://github.com/cybervizay/first4" rel="nofollow noopener">https://github.com/cybervizay/first4</a></p>

**Comment by jorenbroekema** [comment]
<p>I’m having the same problem <a href="https://github.com/jorenbroekema/modern-monorepo-test/actions" rel="nofollow noopener">https://github.com/jorenbroekema/modern-monorepo-test/actions</a></p>
<p>I’ve clicked enable many times, but it keeps going back to being disabled again. Really weird. I have it working properly on a different repo just fine.</p>

**Comment by etx** [comment]
<p>I’m also having this issue on a Private repo.</p>

**Comment by agramfort** [comment]
<p>having the same problem with <a href="https://github.com/mne-tools/mne-bids/pull/430" rel="nofollow noopener">https://github.com/mne-tools/mne-bids/</a></p>

**Comment by CoderSpinoza** [comment]
<p>This is also happening to me for one of my repo (<a href="https://github.com/kakao/kakao_flutter_sdk" rel="nofollow noopener">https://github.com/kakao/kakao_flutter_sdk</a>). This is a transferred repository.</p>

**Comment by DimaRU** [comment]
<p>I have the same issue.</p>
<p><a href="https://github.com/DimaRU/TridentCockpit/actions" rel="nofollow noopener">https://github.com/DimaRU/TridentCockpit/actions</a></p>

**Comment by uesteibar** [comment]
<p>Having the same issue with <a href="https://github.com/uesteibar/lainoa" rel="nofollow noopener">https://github.com/uesteibar/lainoa</a>, it’s a renamed repository</p>

**Comment by carlomazzaferro** [comment]
<p>Same issues here, no matter the configuration of the repo (private, public, on events, etc.). Example: <a href="https://github.com/carlomazzaferro/test-project" rel="nofollow noopener">https://github.com/carlomazzaferro/test-project</a></p>

**Comment by aardappel** [comment]
<p>This is still happening for me today, and not related to transferred repos. See <a href="https://github.com/aardappel/procrastitracker/actions" rel="nofollow noopener">https://github.com/aardappel/procrastitracker/actions</a> that clearly has a workflow file in the repo: <a href="https://github.com/aardappel/procrastitracker/blob/master/.github/workflows/build.yml" rel="nofollow noopener">https://github.com/aardappel/procrastitracker/blob/master/.github/workflows/build.yml</a></p>

**Comment by kjjanak** [comment]
<p>Same Issue - Private Repo</p>

**Comment by Samehadar** [comment]
<p>Hi, actions doesnt work for this repo too <a href="https://github.com/Samehadar/manga-library" rel="nofollow noopener">https://github.com/Samehadar/manga-library</a></p>
<p>I guess, the same problem</p>

**Comment by KaiserNicols** [comment]
<p>Also having this issue with a private repo.</p>

**Comment by joinsalut** [comment]
<p>I don’t think this is solved. Having an issue on private repo.</p>

**Comment by bobmagicii** [comment]
<p>soz, didnt realize that was the wrong reply btn</p>

**Comment by bobmagicii** [comment]
<p>same issue on my public repo <a href="https://github.com/netherphp/avenue/actions" rel="nofollow noopener">https://github.com/netherphp/avenue/actions</a></p>
<p>variation though, it says actions enabled successful, but then going back to actions is the green button again. and they are never getting run.</p>

**Comment by dendroman** [comment]
<p>Same issue on my private repo. Actions stays disabled</p>

**Comment by ssrobins** [comment]
<p>Glad I saw this before spending too much time trying to get my first GitHub Action working.</p>
<p>Since everyone else is doing it, same problem here: <a href="https://github.com/ssrobins/actions-fiddling/" rel="nofollow noopener">https://github.com/ssrobins/actions-fiddling/</a></p>

**Comment by Altify-Development** [comment]
All of my repositories on privated and public have actions disabled, GitHub says to make a ticket, but when I did, they didn't answer and even on my organization, I can't enable actions/workflows. Someone please help.

**Comment by JackB7145** [comment]
Same issue

**Comment by TTaburou** [comment]
I have same issue.
https://github.com/TTaburou/GitHubAcitonsTrial

**Comment by ghost** [comment]
Same issue, but it works on another account. Maybe this one is to young or has taken some kind of flags with the signup process so it can not use workflow actions?

**Reply by diogenesderm2** [reply]
Eu estava com o mesmo problema e so resolvi criando uma conta nova e migrando meus projetos para essa nova conta. E rodou as actions normalmente.

**Comment by davidacoltzi** [comment]
Same issue in my account. Does anyone know how to solve the problem?

**Reply by andremillet** [reply]
same here

**Comment by seifaibrid** [comment]
We have the same issue with all of our Org repos. It does not say why or how to fix it. Did anyone manage to resolve it?

**Comment by yonessElM** [comment]
i have same issue
